<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1177</width>
    <height>941</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab 03</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="3" column="0">
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MODEL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="model_choose">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_model">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DELETE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load_button">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>LOAD</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>OY</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="oy_box">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <double>-10000000000000000000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="ox_box">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <double>-10000000000000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="oz_box">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <double>-99999999999999991611392.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>OZ</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="turn_button">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TURN</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>OX</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="move_label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="dz_box">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <double>-99999999999999991611392.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="dx_box">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <double>-10000000000000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DY</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="dy_box">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <double>-10000000000000000000000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DZ</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="move_button">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MOVE</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DX</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="8" colspan="7">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QFrame" name="frame_5">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CAMERA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="camera_choose">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_camera">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ADD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_camera">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DELETE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" rowspan="3">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QPushButton" name="down_button">
        <property name="font">
         <font>
          <family>Liberation Sans</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>↓</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="up_button">
        <property name="font">
         <font>
          <family>Liberation Sans</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>↑</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="left_button">
        <property name="font">
         <font>
          <family>Liberation Sans</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>←</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="right_button">
        <property name="font">
         <font>
          <family>Liberation Sans</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>→</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="kx_box">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="scale_button">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SCALE</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>KY</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="ky_box">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>KZ</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="kz_box">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000000000000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>KX</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="3" colspan="3">
     <widget class="QPushButton" name="clear_screen">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Liberation Sans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CLEAN SCREEN</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>dx_box</tabstop>
  <tabstop>dy_box</tabstop>
  <tabstop>dz_box</tabstop>
  <tabstop>move_button</tabstop>
  <tabstop>kx_box</tabstop>
  <tabstop>ky_box</tabstop>
  <tabstop>kz_box</tabstop>
  <tabstop>scale_button</tabstop>
  <tabstop>ox_box</tabstop>
  <tabstop>oy_box</tabstop>
  <tabstop>oz_box</tabstop>
  <tabstop>turn_button</tabstop>
  <tabstop>model_choose</tabstop>
  <tabstop>delete_model</tabstop>
  <tabstop>camera_choose</tabstop>
  <tabstop>delete_camera</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
